src/client/common/components/base-network-view/config/menus/file-download.js

```diff
@@ -7,8 +7,8 @@ const AsyncButton = require('../../../async-button');
 
 const { ServerAPI } = require('../../../../../services/');
 
-const downloadTypes = require('../../../../config').downloadTypes;
-
+const downloadTypesFull = require('../../../../config').downloadTypes;
+const downloadTypes = location.pathname.includes('interactions') ? downloadTypesFull.filter(option=> option.type==='png'||option.type==='sif'):downloadTypesFull ;
```

Comment:
you will have to find a way to inject download types as a dependency and specify them like how to component config does.

Alternatively you could specify a custom file download type menu and bind that to the file download button.  There are many ways to do this, we should talk in person.